SetCommonTools.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+
+:: prefer building with Dev15
+
+set CommonTools=%ProgramFiles(x86)%\Microsoft Visual Studio 15.0\Common7\Tools\
```

Comment:
`CommonTools` seems like a variable someone might have defined locally. Can we do something like `RoslynVsCommonTools` or something more specific?